### ISSUE TITLE
JSON output doesn't include most dates

### ISSUE THREAD

**Issue by erdem-iptv** [mkrufky/libdvbtee#51]
I can see the programs on the console output as a list but most of them don't include in the JSON output. It is like doesn't include this list in the JSON part.
The command I enter
`dvbtee/dvbtee -iudp://239.0.0.4:1234 -t30 -j`

and the [output.txt](https://github.com/mkrufky/libdvbtee/files/3480298/output.txt)

**Comment by mkrufky** [comment]
There are `startTime` and `lengthSec` fields, along with `unixTimeBegin` and `unixTimeEnd` fields present in your sample output.

**Comment by erdem-iptv** [comment]
Yes but the problem is JSON data starts from the date 2019-08-24. The program data before that date is not included in the JSON data.

**Comment by mkrufky** [comment]
That does sound like a bug.  I can take a look over the weekend -- do you have a sample TS that I can download from you to test?

**Comment by erdem-iptv** [comment]
I just captured the ts file
https://drive.google.com/open?id=1LRQ74Rjf-iHPG8dtzl1wzV2wMAcS75_f

Many thanks for the support!

**Comment by erdem-iptv** [comment]
Hello mkrufky,
Is there a chance to fix this bug any soon?
Many thanks in advance.

**Comment by mkrufky** [comment]
My apologies - I haven't had a chance to reproduce this yet.  I said that I would take a look almost 2 weeks ago but life happens :-/ . I'll try to find some time for it over the next few days - Please keep the sample file available.

Without looking, however, one possibility comes to mind - Are you sure the missing programs are on the same mux / service that you've recorded?   Often there will be programs listed from other services broadcast due to shared NIT tables, I'm not sure if dvbtee shows them all by default.

Just some food for thought.  I'll take a look at the sample and see what's really happening.

**Comment by erdem-iptv** [comment]
I compare the output with the VLC player program guide,  the output listed all programs in the channel but the only problem is not including this all program list into the JSON part.

Programs list correctly in the part of the output

`store EIT-255: v22 | ts_id 10304 | network_id 1 service_id 10304 | table id: 0x50, last_table id: 0x55
  02:35 - 04:00 : Die Blücherbande
  04:00 - 04:50 : Vorstadtweiber (7/10)
  04:50 - 05:40 : Heiter bis tödlich: Alles Klara (46)
  05:40 - 06:00 : Es geschah an der Grenze (2)
  06:00 - 06:27 : SWR Aktuell Rheinland-Pfalz
  06:27 - 06:30 : Rheinland-Pfalz Wetter
  06:30 - 06:57 : SWR Aktuell Baden-Württemberg
...
`

and

`store EIT-255: v29 | ts_id 10304 | network_id 1 service_id 10304 | table id: 0x51, last_table id: 0x55
  02:15 - 02:45 : So lacht der Südwesten - extra
  02:45 - 03:30 : Das Beste aus \"Rhein in Flammen\"
  03:30 - 04:15 : Spätschicht - Die SWR Comedy Bühne
  04:15 - 04:45 : So lacht der Südwesten
  04:45 - 05:15 : So lacht der Südwesten - extra
  05:15 - 06:00 : Erik Lehmann in kabarett.com
...
`

it continues like that and I can see all the programs. 
I think it is a very small bug that not including all at the JSON part.

**Comment by erdem-iptv** [comment]
Is there any news about this issue?
